### ISSUE TITLE
Timers: add support for "half" and "quarter"

### ISSUE THREAD

**Issue by davefx** [home-assistant/intents#2249]
At this point we cannot create a timer for "half an hour" or for "3 hours and a half".
In Spanish this is a very common case of use, not only for hours, but also for minutes.

I think currently we cannot build these kind of sentences.

**Comment by synesthesiam** [comment]
Hi @davefx! This is possible using lists as shown here: https://github.com/home-assistant/intents/pull/2252

I have a list like this:
```yaml
  timer_half:
    values:
      - in: "half"
        out: 30
```

which can then be used in a sentence like `start a timer for {timer_half:minutes} an hour".

Will this work for Spanish?

**Comment by davefx** [comment]
Yes, this definitely works.
Closed by #2265
